roles/mysql/handlers/main.yml

```
@@ -0,0 +1,27 @@
+---
+# handlers file for mysql
+#
+- name: systemctl daemon-reexec
```

How about this as a name?
```suggestion
- name: restart systemd without rebooting the entire system
```